packages/commercetools/theme/exampleCache.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+
+const CacheDriver = (options, invalidators) => {
+  console.log('driver called', options);
+
+  return {
+    invoke: async ({ getTags, render }) => {
+
+      const res = await render();
```

Comment:
What's the format returned from this function? I mean the cache driver should have full access to the rendered body + headers in order to store it - then when it's a hit to set the headers from cache. Some drivers (like AKAMAI) won't store anything but just operate on the headers (setting proper cache headers)

Reply:
@pkarw invoke has the following signature:
```
async ({  route, context, getTags, render })
```

inside of the context you have access to `req` and `res` which means you have full access to HTTP headers, so you can modyfi them eg. by adding your own headers or even manipulate the response (depending on the needs)

Reply:
when it comes to `render` it mostly returns rendered markup